src/ci/github-actions/ci.yml

```diff
@@ -153,6 +153,10 @@ x--expand-yaml-anchors--remove:
         run: src/ci/scripts/install-clang.sh
         <<: *step
 
+      - name: select Xcode version
```

Comment:
This step needs to be above `install-clang.sh` to make `xcrun --sdk macosx --show-sdk-path` work.

Reply:
That line is disabled because it’s passed in via the environment variable.

Reply:
Yep. Still, running it beforehand could prevent surprising behavior in the future.